elyra/templates/airflow/airflow_template.jinja2

```diff
@@ -16,6 +19,19 @@ dag = DAG(
     is_paused_upon_creation={{ is_paused_upon_creation }},
 )
 
+{% if cos_secret %}
+env_var_secret_id = Secret(deploy_type='env',
```

Comment:
Should add a comment before this line indicating that this secret must be defined manually in the user namespace where the pipeline is run.